guides/common/modules/proc_customizing-the-registration-templates.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Customizing_the_Registration_Templates_{context}"]
+= Customizing the Registration Templates
+
+You can customize the registration process by editing the provisioning templates.
+Note that all default templates in {Project} are locked.
+If you want to customize the registration templates, you must clone the default templates and edit the clones.
+
+The registration process uses the following provisioning templates:
+
+* The `Global Registration` template contains steps for registering hosts to {Project}.
```

Comment:
```suggestion
* The *Global Registration* template contains steps for registering hosts to {Project}.
```

Reply:
I'm against this, because we use preformatted style for template names elsewhere.

Reply:
Works for me!